### PR TITLE
What's new in the .NET Standard 2.0

### PR DESCRIPTION
**Pull request by rpetrusha**
# What's new in the .NET Standard 2.0

## Suggested Reviewers

@BillWagner, @mairaw, @terrajobst

### CODE REVIEW

File: docs/standard/whats-new/whats-new-in-dotnet-standard.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+ 
+The .NET Standard 2.0 includes the following new features:
+
+**A vastly expanded set of APIs**
+
+Through version 1.6, the .NET Standard included a comparatively small subset of APIs. Among those excluded were many APIs that were commonly used in the .NET Framework or Xamarin. This complicates development, since it requires that developers find suitable replacements for familiar APIs when they develop applications and libraries that target multiple .NET implementations. The .NET Standard 2.0 addresses this limitation by adding over 20,000 more APIs than were available in .NET Standard 1.6, the previous version of the standard. For a list of the APIs that have been added to the .NET Standard 2.0, see [.NET Standard 2.0 vs 1.6](https://raw.githubusercontent.com/dotnet/standard/master/docs/versions/netstandard2.0_diff.md). 
+
+Some of the more noteworthy additions to the <xref:System> namespace alone in .NET Standard 2.0 include the following:
+
+- Support for the <xref:System.AppDomain>] class.
```
Comment (JRAlexander) [inline docs/standard/whats-new/whats-new-in-dotnet-standard.md]: Remove the ']'.

File: docs/standard/whats-new/whats-new-in-dotnet-standard.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+ms.technology: dotnet-standard
+ms.##devlang: dotnet
+author: "rpetrusha"
+ms.author: "ronpet"
+manager: "wpickett"
+---
+
+# What's new in the .NET Standard
+
+The .NET Standard is a formal specification that defines a versioned set of APIs which must be available on .NET implementations that comply with that version of the standard. The .NET Standard is targeted at library developers. It aims to guarantee the functionality that is available to a library that targets a version of the .NET Standard on each .NET implementation. In other words, a library that targets a version of the .NET Standard can be used on any .NET Framework, .NET Core, or Xamarin implementation that supports that version of the standard.
```
Comment (JRAlexander) [inline docs/standard/whats-new/whats-new-in-dotnet-standard.md]: I would change "It aims to guarantee the functionality that is available to a library that targets a version of the .NET Standard on each .NET implementation. In other words, a library that targets a version of the .NET Standard can be used on any .NET Framework, .NET Core, or Xamarin implementation that supports that version of the standard." to something like "A library that targets a .NET Standard version can be used on any .NET Framework, .NET Core, or Xamarin implementation that supports that standard version."

File: docs/standard/whats-new/whats-new-in-dotnet-standard.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+## What's new in the .NET Standard 2.0
+ 
+The .NET Standard 2.0 includes the following new features:
+
+**A vastly expanded set of APIs**
+
+Through version 1.6, the .NET Standard included a comparatively small subset of APIs. Among those excluded were many APIs that were commonly used in the .NET Framework or Xamarin. This complicates development, since it requires that developers find suitable replacements for familiar APIs when they develop applications and libraries that target multiple .NET implementations. The .NET Standard 2.0 addresses this limitation by adding over 20,000 more APIs than were available in .NET Standard 1.6, the previous version of the standard. For a list of the APIs that have been added to the .NET Standard 2.0, see [.NET Standard 2.0 vs 1.6](https://raw.githubusercontent.com/dotnet/standard/master/docs/versions/netstandard2.0_diff.md). 
+
+Some of the more noteworthy additions to the <xref:System> namespace alone in .NET Standard 2.0 include the following:
+
```
Comment (JRAlexander) [inline docs/standard/whats-new/whats-new-in-dotnet-standard.md]: I would replace with something like "<xref:System> namespace additions in .NET Standard 2.0 include:".

File: docs/standard/whats-new/whats-new-in-dotnet-standard.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+The overwhelming majority of libraries target the .NET Framework rather than .NET Standard. However, most of the calls in those libraries are to APIs that are included in the .NET Standard 2.0. Starting with the .NET Standard 2.0, you can access .NET Framework libraries from a .NET Standard library by using a [compatibility shim](https://github.com/dotnet/standard/blob/master/docs/netstandard-20/README.md#assembly-unification). This compatibility layer is transparent to developers; you don't have to do anything to take advantage of .NET Framework libraries.
+
+The single requirement is that the APIs called by the .NET Framework class library must be included in the .NET Standard 2.0.
+
+**Support for Visual Basic**
+
+You can now develop .NET Standard libraries in Visual Basic. For Visual Basic developers using Visual Studio 2017, Visual Studio now includes a .NET Standard Class Library template. For Visual Basic developers who use other development tools and environments, you can use the [dotnet new](../../core/tools/dotnet-new.md) command to create a .NET Standard Library project. For more information, see the [Tooling support for .NET Standard libraries](#tooling).
```
Comment (JRAlexander) [inline docs/standard/whats-new/whats-new-in-dotnet-standard.md]: Is there a minimum version of Visual Studio needed for the .NET Standard Template?

**Review by JRAlexander — CHANGES_REQUESTED** [review]
LGTM. Overall, great job, @rpetrusha! Just a few small things.

**Comment by rpetrusha** [comment]
Thanks for the review, @JRAlexander. I've addressed all of your comments.

File: docs/standard/whats-new/whats-new-in-dotnet-standard.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+## What's new in the .NET Standard 2.0
+ 
+The .NET Standard 2.0 includes the following new features:
+
+**A vastly expanded set of APIs**
+
+Through version 1.6, the .NET Standard included a comparatively small subset of APIs. Among those excluded were many APIs that were commonly used in the .NET Framework or Xamarin. This complicates development, since it requires that developers find suitable replacements for familiar APIs when they develop applications and libraries that target multiple .NET implementations. The .NET Standard 2.0 addresses this limitation by adding over 20,000 more APIs than were available in .NET Standard 1.6, the previous version of the standard. For a list of the APIs that have been added to the .NET Standard 2.0, see [.NET Standard 2.0 vs 1.6](https://raw.githubusercontent.com/dotnet/standard/master/docs/versions/netstandard2.0_diff.md). 
+
+Some of the more additions to the <xref:System> namespace in .NET Standard 2.0 include:
```
Comment (JRAlexander) [inline docs/standard/whats-new/whats-new-in-dotnet-standard.md]: This looks like a typo.

**Comment by rpetrusha** [comment]
Thanks for catching the typo, @JRAlexander. I've corrected it.

**Review by JRAlexander — APPROVED** [review]

**Comment by JRAlexander** [comment]
LGTM. Great work, @rpetrusha!

**Comment by rpetrusha** [comment]
closing and reopening to begin new build...

**Pull request merged**